test_conformance/device_execution/enqueue_flags.cpp

```diff
@@ -82,7 +82,6 @@ static const char* enqueue_flags_wait_kernel_simple[] =
     NL, "  size_t gs  = get_global_size(0);"
     NL, "  size_t tid  = get_global_id(0);"
     NL, ""
-    NL, "  res[tid] = 0;"
     NL, "  array[tid] = tid;"
```

Comment:
It would actually be better to move this after the enqueue.

Reply:
This will not resolve the data race condition.

Reply:
Agreed, but it would improve the test.

Reply:
Ok, done